app/routes/routes.js

```diff
@@ -12,9 +12,9 @@ module.exports.setup = function setup(app, serviceLocator) {
     app.get('/admin/',  function (req, res, next) {
         res.render('admin_dashboard', {layout : 'admin_main'});
     });
-
     app.get('/admin/add-rhapsody-realities',  (req, res, next) => rhapsodyController.addNewView(req, res, next));
     app.post('/admin/add-rhapsody-realities',(req, res, next) => rhapsodyController.save(req, res, next));
-
     app.get('/admin/list-rhapsody-realities',  (req, res, next) => rhapsodyController.list(req, res, next));
+    app.get('/admin/edit-rhapsody-realities/:id',  (req, res, next) => rhapsodyController.editView(req, res, next));
+    app.post('/admin/edit-rhapsody-realities',  (req, res, next) => rhapsodyController.update(req, res, next));
```

Comment:
`post` will work, but can you see if there is a `put` ?
we usually use `put` for update, although very common in api only projects, wonder if express has something like that

Reply:
PUT verb was not use any where in the route of the ref project